api_docgen/odoc/Makefile

```
@@ -137,17 +143,19 @@ stdlib_INDEX=\
   $(foreach m,$(stdlib_UNPREFIXED),$(call stdlib_prefix,$m))\
   $(call capitalize, $(otherlibref))
 build/libref.mld:
-	echo {0 OCaml standard library} {!modules:$(stdlib_INDEX)} > $@
+	( echo "{0 OCaml standard library}"; \
+    echo "{!modules:$(stdlib_INDEX)}" ) > $@
```

indentation is wrong
```suggestion
	( echo "{0 OCaml standard library}"; \
	  echo "{!modules:$(stdlib_INDEX)}" ) > $@
```